### PR TITLE
[OSCI] Add new author brianzheng

### PR DESCRIPTION
**Pull request by Frederic-Chopin**
### Description
Added brianzheng.markdown 
 
### Issues Resolved
N/A

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/brianzheng.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Brian Zheng
+short_name: brianzheng
+twitter: Brian48763045
+github: Frederic-Chopin
+---
+
+**<Name>** I'm a senior studying CS at UC Berkeley. I love to play soccer, go hiking, and play poker!
```
Comment (cwperks) [inline _authors/brianzheng.markdown:8]: Please insert your name on this line in the `<Name>` placeholder.

**Review by cwperks — CHANGES_REQUESTED** [review]
Please replace the `<Name>` placeholder. Otherwise this LGTM.

File: _authors/brianzheng.markdown (hunk 1 above)
Comment (Frederic-Chopin) [reply _authors/brianzheng.markdown]: Fixed ty!

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
@Frederic-Chopin Sorry, do you mind moving this file to the [authors/osci_2023](https://github.com/opensearch-project/project-website/tree/main/_authors/osci_2023) directory?

**Comment by Frederic-Chopin** [comment]
Just did! Ty!

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**